core/src/main/scala/replpp/ReplDriver.scala

```diff
@@ -125,8 +124,16 @@ class ReplDriver(args: Array[String],
       // This is analogous to what happens in dotty.tools.repl.Rendering.
       val pprinter = Class.forName("replpp.PPrinter", true, rendering.myClassLoader)
       val renderer = pprinter.getMethod("apply", classOf[Object])
-      (value: Object, maxElements: Int, maxCharacters: Int) =>
-        renderer.invoke(null, value).asInstanceOf[String].take(maxCharacters)
+      (value: Object, maxElements: Int, maxCharacters: Int) => {
+        val rendered = renderer.invoke(null, value).asInstanceOf[String]
+
+        if (maxCharacters > 0 && rendered.length >= maxCharacters) {
+          // same behaviour as in dotty.tools.repl.Rendering
+          s"${rendered.take(maxCharacters)} ... large output truncated, print value to show all"
```

Comment:
is there an easy way here to determine the `resX` variable name for the value here? I feel like something like "...truncated, use print(resX) to show all" would be more clear about what I'm supposed to do. But it's not so important to jump through any hoops to improve this message.